realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+# find javah
+find_package(Java COMPONENTS Development)
+if (NOT Java_Development_FOUND)
+    if (DEFINED ENV{JAVA_HOME} AND EXISTS "$ENV{JAVA_HOME}/bin/javah")
+      set(Java_JAVAH_EXECUTABLE "$ENV{JAVA_HOME}/bin/javah")
+    elseif (EXISTS "/usr/bin/javah")
+      set(Java_JAVAH_EXECUTABLE "/usr/bin/javah")
+    else()
+      message(FATAL_ERROR "Cannot find javah")
+    endif()
+endif()
+include (UseJava)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+create_javah(TARGET jni_headers
+  CLASSES io.realm.internal.Table io.realm.internal.TableView
```

Comment:
Is there a smart way to say that all classes with `native` functions should be included?